global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.101",
+    "version": "8.0.204",
```

Comment:
No need to upgrade this. 
I upgrade this by merging in the Library.Template repo periodically, so doing this directly here would just introduce merge conflicts.